roles/monitoring/defaults/main.yaml

```diff
@@ -37,3 +37,5 @@ V4M_GRAFANA_FQDN: "grafana.{{ V4M_BASE_DOMAIN }}"
 V4M_GRAFANA_CERT: "{{ V4M_CERT }}"
 V4M_GRAFANA_KEY: "{{ V4M_KEY }}"
 V4M_GRAFANA_PASSWORD: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}"
+
+V4_CFG_TLS_GENERATOR: openssl # [cert-manager,openssl]
```

Comment:
Why is this here? It's a generic V4 variable and should already be defined with a value. This should not be added here. You'll have to verify that the variable you're looking for is defined properly and has a default. This is not the place.